todo_app/templates/index.html

```diff
@@ -7,11 +7,31 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <div class="mb-5">
+    <h2 class="font-weight-light">Add To-Do Item</h2>
+    <form action="/" method="post">
+        <input type="text" class="form-control mb-1" name="add-item">
```

Comment:
If we added a `required` tag to this input, our browser can help prevent users from submitting e.g. an empty field. If this is vital we should obviously also check on the backend, but here it might be reasonable to accept that risk but help users try to avoid it